pytensor/printing.py

```diff
@@ -374,7 +383,7 @@ def debugprint(
             _debugprint(
                 ig_var,
                 prefix=prefix,
-                depth=depth,
+                depth=inner_depth,
```

Comment:
depth and inner_depth are not the same. Depth still makes sense alongside inner_depth. Inner depth tells how many inner graphs to step into, depth tells how many ops in a graph (or inner garph) to step into